### PR TITLE
Update location of package ToricVarieties

### PR DESCRIPTION
**Pull request by HereAround**

### CODE REVIEW

**Comment by alex-konovalov** [comment]
@HereAround thanks but https://github.com/homalg-project/ToricVarieties_project/ToricVarieties/PackageInfo.g is broken link.

**Comment by HereAround** [comment]
> @HereAround thanks but https://github.com/homalg-project/ToricVarieties_project/ToricVarieties/PackageInfo.g is broken link.

Sorry about that. https://github.com/gap-system/gap-distribution/pull/115/commits/0b8d3f89636071d43769113125fb4e0975428527 should fix this.

File: DistributionUpdate/PackageUpdate/currentPackageInfoURLList [hunk 1]
```diff
@@ -160,7 +160,7 @@ Thelma      https://gap-packages.github.io/Thelma/PackageInfo.g
 TomLib      https://gap-packages.github.io/tomlib/PackageInfo.g
 ToolsForHomalg https://homalg-project.github.io/homalg_project/ToolsForHomalg/PackageInfo.g
 toric       https://gap-packages.github.io/toric/PackageInfo.g
-ToricVarieties https://homalg-project.github.io/homalg_project/ToricVarieties/PackageInfo.g
+ToricVarieties MOVE https://homalg-project.github.io/ToricVarieties_project/ToricVarieties/PackageInfo.g
```
Comment (alex-konovalov) [inline DistributionUpdate/PackageUpdate/currentPackageInfoURLList:163]: Thanks @HereAround. Just to check, is it still correct that URLs in
```
SourceRepository := rec( 
  Type := "git", 
  URL := "https://github.com/homalg-project/homalg_project"
),
IssueTrackerURL := Concatenation( ~.SourceRepository.URL, "/issues" ),
PackageWWWHome := "https://homalg-project.github.io/homalg_project/ToricVarieties/",
ArchiveFormats := ".tar.gz .zip",
ArchiveURL     := Concatenation( "https://github.com/homalg-project/homalg_project/releases/download/ToricVarieties-", ~.Version, "/ToricVarieties-", ~.Version ),
```
point to `homalg-project/homalg_project` repository?

Comment (HereAround) [reply DistributionUpdate/PackageUpdate/currentPackageInfoURLList:163]: Thank you for pointing this out. I have updated the PackageInfo.g with the new URL.

**Comment by mohamed-barakat** [comment]
Congratulations Martin and many thanks Alexander.

**Comment by alex-konovalov** [comment]
Not yet @HereAround :)The combination
```
PackageWWWHome := "https://homalg-project.github.io/ToricVarieties_project/ToricVarieties",
PackageInfoURL := Concatenation( ~.PackageWWWHome, "PackageInfo.g" ),
```
does not work due to the missing `/`, so it causes broken link in `PackageInfoURL`.

**Comment by HereAround** [comment]
@alex-konovalov - Thank you for pointing this out. I have added the necessary '/' now into the PackageInfo. I suppose this should work now.

**Comment by alex-konovalov** [comment]
@HereAround thanks, next hurdle is here:
```
* ToricVarieties - new version 2020.08.03 discovered!!!
  ============================================
  Getting new archives from 
  https://homalg-project.github.io/ToricVarieties_project/ToricVarieties/
[ ".zip" ]
  unpacking .zip
Archive:  .zip
ERROR (ToricVarieties): could not unpack archive .... SKIPPING !!!
```

**Comment by HereAround** [comment]
@alex-konovalov Thank you for pointing this out. I have uploaded a .zip with the release of the ToricVarieties_project (https://github.com/homalg-project/ToricVarieties_project/releases/tag/2020-08-16). The PackageInfo has been changed accordingly just now https://github.com/homalg-project/ToricVarieties_project/commit/94606cd5eea0b0f69f336fac3b8a2ec7f959ea71. With these changes, please let us have another go at this.

**Comment by alex-konovalov** [comment]
@HereAround thanks - if all is fine, then it should be automatically picked up tomorrow. Let's see...

**Pull request merged**